pkg/ebpf/c/tracee.bpf.c

```diff
@@ -688,21 +743,25 @@ static __always_inline bool find_modules_from_module_kset_list(program_data_t *p
     return !finished_iterating;
 }
 
-BPF_QUEUE(walk_mod_tree_queue, rb_node_t, 2048); // used to walk a rb tree
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 20, 0) || defined(CORE)
```

Comment:
4,20 ? Either way, we do not support non CORE any longer. So you can remove this ifdef here.